### PR TITLE
updates in gated marketplace and fruniques

### PR DESCRIPTION
**Pull request by tlacloc**
This PR includes changes to the new extrinsics of fruniques as the new spawning method and a new flow of the creation of collections.
Also includes the extrinsics to enlist a buy / sell offer, delete it and accept it.

### CODE REVIEW

File: src/model/polkadot-pallets/afloatApi.js [hunk 1]
```diff
@@ -31,11 +48,10 @@ class AfloatApi extends BasePolkadot {
    * @param {Function} subTrigger Function to trigger when subscrsption detect changes
    * @returns {Object}
    */
-  async createAsset ({ collectionId, assetId, uniquesPublicAttributes, saveToIPFS, cidFromHCD, admin }, subTriger) {
+  async createAsset ({ collectionId, uniquesPublicAttributes, saveToIPFS, cidFromHCD, ParentId, isHierarchical, percentage }, subTriger) {
```
Comment (jmgayosso) [inline src/model/polkadot-pallets/afloatApi.js:51]: please use camel case to parentId

File: src/model/polkadot-pallets/afloatApi.js [hunk 2]
```diff
@@ -31,11 +48,10 @@ class AfloatApi extends BasePolkadot {
    * @param {Function} subTrigger Function to trigger when subscrsption detect changes
```
Comment (jmgayosso) [inline src/model/polkadot-pallets/afloatApi.js:48]: please add @param description for each new param

File: src/model/polkadot-pallets/fruniquesApi.js [hunk 3]
```diff
@@ -4,9 +4,36 @@ class FruniquesApi extends BasePolkadot {
...
   }
+  /**
+   * @name create_collection
+   * @param {String} metadata The metadata of the collection
+   * @returns
+   */
 
-  createWithAttributes ({ user, classId, instanceId, numericValue, admin, attributes }, subTrigger) {
-    return this.callTx('createWithAttributes', user, [classId, instanceId, numericValue, admin, attributes], subTrigger)
+  create_collection ({ user, metadata }, subTrigger) {
```
Comment (jmgayosso) [inline src/model/polkadot-pallets/fruniquesApi.js:13]: Please add description for each param

File: src/model/polkadot-pallets/fruniquesApi.js [hunk 4]
```diff
@@ -4,9 +4,36 @@ class FruniquesApi extends BasePolkadot {
...
+  }
+
+  /**
+   * @name spawn
+   * @param {u32} classId Collection ID used in the uniques pallet; represents a group of Uniques
+   * @param {Tuple} subTrigger Contains information of the parent as (parentId, hierarchicalAttributes, percentageOfParent)
+   * @param {Array} Attributes Array of attributes
+   * @returns
+   */
+  spawn ({ user, classId, parentInfo, Attributes }, subTrigger) {
```
Comment (jmgayosso) [inline src/model/polkadot-pallets/fruniquesApi.js:24]: please use camel case and add description to params

**Review by garcia-alejandro — APPROVED** [review]

**Pull request merged**